server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/ExpressionAnalyzer.java

```diff
@@ -590,6 +723,10 @@ public static Pair<Filter, Boolean> transformToGlobalTimeFilter(Expression predi
         return new Pair<>(timeInRightFilter, false);
       }
       return new Pair<>(null, true);
+    } else if (predicate instanceof LikeExpression || predicate instanceof RegularExpression) {
+      return new Pair<>(null, true);
+    } else if (predicate instanceof BetweenExpression) {
+      return new Pair<>(null, true);
```

Comment:
Do we now support `where time between 1 and 10` syntax? If so, here we should return the corresponding time filter.

Reply:
Yes, we support. Is corresponding time filter used to filter quickly? We can get the same result even if treat it as value filter.